### PR TITLE
Add ability to configure random values (e.g. pajbot_host)

### LINKED ISSUE #33: Add a pajbot_url configure option
Instead of hardcoding forsen's pajbot website into the bot, add a config option to allow users to change it to what they want:

https://github.com/pajbot/pajbot2-discord/blob/e1bf861d7ef356dbc8b599df5bc43a81d895cb57/internal/commands/points/points.go#L47

### PR DESCRIPTION
**Pull request by pajlada**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

- Add a way to configure the `pajbot_host` value used in places like $points
- Update the $points command to use the configurable `pajbot_host` value

Fixes #33

### CODE REVIEW

**Comment by pajlada** [comment]
requesting a review from @alazymeme pajaDink

File: internal/commands/configure/configure.go [hunk 1]
```diff
@@ -59,6 +59,108 @@ var discordEmojiRegex = regexp.MustCompile(`<(a)?:([^<>:]+):([0-9]+)>`)
...
+	case "reset":
+		err := serverconfig.Remove(commands.SQLClient, m.GuildID, dbKey)
+		if err != nil {
+			fmt.Println("Error removing value:", err)
+			return
+		}
+
+		const f = "%s, key %s reset. old value was %s"
+		r := fmt.Sprintf(f, m.Author.Mention(), key, oldValue)
+		s.ChannelMessageSend(m.ChannelID, r)
```
Comment (alazymeme) [inline internal/commands/configure/configure.go:113]: This seems to be appended even when there was no original value. Are we able to not post this portion of the message if there was no `oldValue`?

image

**Comment by alazymeme** [comment]
Other than the above, it works perfectly

File: internal/commands/configure/configure.go (hunk 1 above)
Comment (pajlada) [reply internal/commands/configure/configure.go]: Fixed it in [`3162b66` (#46)](https://github.com/pajbot/pajbot2-discord/pull/46/commits/3162b667ecad861605a2aa6cb7449e9d2f80d99c) :+1:

**Comment by pajlada** [comment]
image

**Pull request merged**